packages/extension-ui/src/components/InputWithLabel.tsx

```diff
@@ -80,20 +85,27 @@ function InputWithLabel({
         placeholder={placeholder}
         readOnly={isReadOnly}
         spellCheck={false}
-        type={type}
+        type={isObscured ? type : 'text'}
         value={value}
         withError={isError}
       />
+      {type === "password" && (
+        <button
+          className='password-icon'
```

Comment:
I didn't want to change it, because I thought I'm gonna overwrite some styles from parents applied to this class, but it doesn't seem so, gonna change it :+1: